lib/cfg-tree.h

```diff
@@ -40,20 +40,20 @@ const gchar *log_expr_node_get_content_name(gint content);
 enum
 {
   /* expr node content type */
-  ENC_SOURCE,
```

Comment:
Just a side note: I would create two different commit: one for the fix (the `.c` file), and one for the refactor (`.h` file).